### PR TITLE
Add derequire standalone build for RequireJS compat.

### PR DESCRIPTION
**Pull request by MadLittleMods**
I tried using the standalone build in a RequireJS environment and realized there are some lingering `define` calls in some of the dependencies that mess up when trying to use it in the r.js optimizer/bundler.

I piped the browserify output into [derequire](https://www.npmjs.com/package/derequire) and save it out like normal.

### CODE REVIEW

**Comment by pocketjoso** [comment]
A little bit ugly, but sure. Can you please rebase your PR on the latest `master`?

**Comment by MadLittleMods** [comment]
Done. 

I wish it could be more concise but the derequire CMD options only allow one replace at a time so I have to pipe it twice.

File: package.json [hunk 1]
```diff
@@ -37,11 +37,13 @@
...
   "devDependencies": {
-    "browserify": "^6.2.0"
+    "browserify": "^6.2.0",
+    "derequire": "^1.2.0"
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "example": "browserify example.js -o dist/bundle.js && open index.html",
-    "build_standalone": "browserify ./lib/core.js --standalone specificityGraph > specificity-graph-standalone.js"
+    "build_standalone": "browserify example.js -o dist/bundle.js && open index.html",
```
Comment (pocketjoso) [inline package.json]: This looks like a mistake?

Comment (MadLittleMods) [inline package.json]: Sorry, was messing with rebase, merge etc. Should be fixed now.

**Pull request merged**